publish/bin/publish-batch.js

```diff
@@ -60,13 +60,13 @@ try {
     client,
     web3Storage,
     ieContract,
-    recordTelemetry,
+    recordPublishTelemetry,
```

Comment:
This rename doesn't look right to me. The `publish` function expects `recordTelemetry` argument. 


Alternatively, we could rename the argument. I would find it a bit repetitive to have a `publish` function with `recordPublishTelemetry` param. I prefer the current version more.

```suggestion
    recordTelemetry: recordPublishTelemetry,
```

Reply:
Addressed in https://github.com/filecoin-station/spark-api/pull/339/commits/6332977b75dd30bd098b3c751b5efbefb59a2d0e